entries/a/aussiebroadband.com.au.json

```
@@ -1,10 +1,13 @@
 {
   "Aussie Broadband": {
     "domain": "aussiebroadband.com.au",
-    "contact": {
-      "facebook": "aussiebroadband",
-      "twitter": "Aussie_BB"
-    },
+    "tfa": [
+      "sms",
+      "email",
+      "totp"
+    ],
+    "documentation": "https://www.aussiebroadband.com.au/help-centre/business/6-carbon/how-to-enable-2-factor-authentication/",
+    "notes": "SMS & Email 2FA are always enabled, setup not required.",
```

The second part is redundant.
```suggestion
    "notes": "SMS & Email 2FA are always enabled.",
```